Description: Do not use Intersphinx plugin in sphinx doc
 This avoids network access.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: Debian-specific
Last-Update: 2014-06-01

--- python-itsdangerous.orig/docs/conf.py
+++ python-itsdangerous/docs/conf.py
@@ -13,7 +13,6 @@
 master_doc = "index"
 extensions = [
     "sphinx.ext.autodoc",
-    "sphinx.ext.intersphinx",
     "pallets_sphinx_themes",
     "sphinxcontrib.log_cabinet",
     "sphinx_issues",
